hrrecruitment/CRM/HRRecruitment/Upgrader.php

```diff
@@ -5,115 +5,57 @@
  */
 class CRM_HRRecruitment_Upgrader extends CRM_HRRecruitment_Upgrader_Base {
 
-  // By convention, functions that look like "function upgrade_NNNN()" are
-  // upgrade tasks. They are executed in order (like Drupal's hook_update_N).
-
-  /**
-   * Example: Run an external SQL script when the module is installed
-   *
-  public function install() {
-    $this->executeSqlFile('sql/myinstall.sql');
-  }
-
   /**
-   * Example: Run an external SQL script when the module is uninstalled
+   * Sets the weight on "Application" CaseType
    *
-  public function uninstall() {
-   $this->executeSqlFile('sql/myuninstall.sql');
-  }
-
-  /**
-   * Example: Run a simple query when a module is enabled
-   *
-  public function enable() {
-    CRM_Core_DAO::executeQuery('UPDATE foo SET is_active = 1 WHERE bar = "whiz"');
-  }
-
-  /**
-   * Example: Run a simple query when a module is disabled
-   *
-  public function disable() {
-    CRM_Core_DAO::executeQuery('UPDATE foo SET is_active = 0 WHERE bar = "whiz"');
+   * @return bool
+   */
+  public function upgrade_1400() {
+    $this->ctx->log->info('Applying update 1400');
+    CRM_Core_DAO::executeQuery("UPDATE civicrm_case_type SET weight = 7 WHERE name = 'Application'");
+    return TRUE;
   }
 
   /**
-   * Example: Run a couple simple queries
-   *
-   * @return TRUE on success
-   * @throws Exception
-   *
-  public function upgrade_4200() {
-    $this->ctx->log->info('Applying update 4200');
-    CRM_Core_DAO::executeQuery('UPDATE foo SET bar = "whiz"');
-    CRM_Core_DAO::executeQuery('DELETE FROM bang WHERE willy = wonka(2)');
-    return TRUE;
-  } // */
-
+   * Remove the "Application" assignment type and the managed entity.
+   *
+   * @return bool
+   */
+  public function upgrade_1402() {
```

Comment:
ah, maybe I wasn't clear. I understood that we would need to execute SQL before upgrading because of the bug you found. It was always my plan to have the upgrader to remove the entities, but I just wanted to check if we needed to execute SQL or not.